src/autocomplete.js

```diff
@@ -119,13 +123,15 @@
         }
 
         initializeEventHandlers() {
+            this.$input[0].addEventListener(this.options.onFocus, this._onFocusHandler);

```

Comment:
If "openOnInput" is true, add handler like this:
`this.$input[0].addEventListener('input', this._onInputHandler);`